src/App.tsx

```diff
@@ -1,25 +1,68 @@
-import React, { useState } from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
 
+import { getTodos } from './api';
+
 const App: React.FC = () => {
-  const [
-    selectedUserId,
-    // setSelectedUserId,
-  ] = useState(0);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectedUserId, setSelectedUserId] = useState(0);
+  const [query, setQuery] = useState('');
+  const [selectedValue, setSelectedValue] = useState('all');
+
+  useEffect(() => {
+    getTodos().then(setTodos);
```

Comment:
there’s a special syntax to work with promises in a more comfortable fashion - async await